src/extensions/screenshooter.rs

```diff
@@ -0,0 +1,27 @@
+use wayland_sys::server::wl_display as wl_server_display;
+use wlroots_sys::{wl_display, wlr_screenshooter_manager, wlr_screenshooter_manager_create, 
+    wlr_screenshooter_manager_destroy};
```

Comment:
Looks like you have the wrong name of functions, they should not have the `manager` in the name.